src/Orchard/Tasks/Locking/Migrations/FrameworkMigrations.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using Orchard.Data.Migration;
+
+namespace Orchard.Tasks.Locking.Migrations {
+    public class FrameworkMigrations : DataMigrationImpl {
+
+        public int Create() {
+            SchemaBuilder.CreateTable("LockRecord", table => table
+                .Column<int>("Id", column => column.PrimaryKey().Identity())
+                .Column<string>("Name", column => column.NotNull().WithLength(256))
+                .Column<string>("Owner", column => column.WithLength(256))
+                .Column<int>("ReferenceCount")
+                .Column<DateTime>("CreatedUtc")
```

Comment:
@Piedone The lock is reentrant, i.e. the same owner can make multiple acquisitions. Look at line 121.

As to why "Count" is better, IMO the record itself represents an acquisition, so "Count" is intrinsic. But now we're really talking semantics.